classroom.rb

```diff
@@ -14,24 +14,36 @@ def add_student
 end
 
 class Book
+  @@books = []
   def initialize(title, author)
     @title = title
     @author = author
+    @@books.push({ 'title' => @title, 'author' => @author })
   end
   attr_accessor :title, :author
 
   def add_rental(date)
     new.Rental(date)
   end
+
+  def list_of_books
+    @@books.each_with_index do |book, idx|
+      puts "#{idx}) Title: #{book['title']}, Author: #{book['author']}"
+    end
+  end
```

Comment:
- [ ] In OOP concept, this method doesn't belong here. The reason behind is this. A class is a blue print for an object. In this cases the blue print represents a book. Now, the question we need to add is does `list_of_books `define a behavior of a book object? I don't it is. Because, a book object is singular but the method deals about list of books - which is plural. So, where do we have a Class that handles list of books? In this project, that class is the App class.